CMakeLists.txt

```diff
@@ -34,6 +34,11 @@ SET(_d2_build_types "Release;Debug")
 SET(Release_postfix "")
 SET(Debug_postfix ".g")
 
+find_package(Boost)
+if(Boost_FOUND)
+  add_definitions(-DDEAL_II_SAK_WITH_BOOST)
+  SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -I$ENV{BOOST_DIR}/include -L$ENV{BOOST_DIR}/lib -lboost_filesystem -lboost_system" )
```

Comment:
You should use the variables which are set by `find_package(boost);` this won't work unless you set `$BOOST_DIR`, which is not mandatory, even if you have installed boost, and cmake could find it...